domains/client/domain-executor/src/domain_worker.rs

```diff
@@ -244,8 +236,8 @@ where
 /// 1. Extract the bundles from the block given the `domain_id`.
 /// 2. Pass the bundles to the respective domain bundle processor and do the computation there.
 async fn process_primary_block<Block, PBlock, PClient, ProcessorFn>(
-    domain_id: DomainId,
-    primary_chain_client: &PClient,
+    _domain_id: DomainId,
```

Comment:
do we need the args if we are ignoring them ?

Reply:
The whole function was eliminated in the later commits.